setup.cfg

```diff
@@ -30,8 +30,7 @@ package_dir =
 packages = find:
 python_requires = >=3.8
 install_requires =
-	qiskit>=0.34.2,<1.0
-	qiskit-terra>=0.19.2,<1.0
+	qiskit>=1.0.0
```

Comment:
just for safety, could we set the upper limit on the qiskit like `qiskit>=1.0.0,<2.0